lib/settings.js

```diff
@@ -7,6 +7,11 @@ settings.s3Key = process.env.S3_KEY;
 settings.s3Secret = process.env.S3_SECRET;
 settings.s3Bucket = process.env.S3_BUCKET;
 
+// Postgresql parcel server
+// Use Heroku-style primary postgresql database environment variable
```

Comment:
I think the Heroku-style environment variable is `DATABASE_URL`. But with the addon attachments system, I believe we can make another app's postgres DB available to the tileserver under a custom name, like `FEATURES_DB`.